src/openvic-simulation/pop/PopValuesFromProvince.cpp

```diff
@@ -0,0 +1,56 @@
+#include "PopValuesFromProvince.hpp"
+#include <cstdint>
+
+#include "openvic-simulation/country/CountryInstance.hpp"
+#include "openvic-simulation/defines/PopsDefines.hpp"
+#include "openvic-simulation/modifier/ModifierEffectCache.hpp"
+#include "openvic-simulation/map/ProvinceInstance.hpp"
+
+using namespace OpenVic;
+
+void PopStrataValuesFromProvince::update(PopValuesFromProvince const& parent, Strata const& strata) {
+	ProvinceInstance const& province = parent.get_province();
+	ModifierEffectCache const& modifier_effect_cache = province.get_modifier_effect_cache();
+	ModifierEffectCache::strata_effects_t const& strata_effects = modifier_effect_cache.get_strata_effects()[strata];
+	PopsDefines const& defines = parent.get_defines();
+	fixed_point_t shared_base_needs_scalar = defines.get_base_goods_demand()
+		* province.get_modifier_effect_value(*modifier_effect_cache.get_goods_demand());
+
+	fixed_point_t invention_needs_scalar = fixed_point_t::_1();
+	CountryInstance const* const owner_nullable = province.get_owner();
+	if(owner_nullable != nullptr) {
+		CountryInstance const& owner = *owner_nullable;
+		shared_base_needs_scalar *= fixed_point_t::_1() + owner.get_plurality();
```

Comment:
```suggestion
		shared_base_needs_scalar *= fixed_point_t::_1() + owner.get_plurality() / 100;
```
`owner.get_plurality` is stored on a 0-100 range rather than 0.00-1.00, so you'll need to normalise it before adding it to `1`